docs/Configuration.md

```diff
@@ -1018,6 +1018,31 @@ function testRunner(
 
 An example of such function can be found in our default [jasmine2 test runner package](https://github.com/facebook/jest/blob/master/packages/jest-jasmine2/src/index.js).
 
+### `testSequencer` [string]
+
+Default: `@jest/test-sequencer`
+
+This option allows you to use a custom sequencer instead of Jest's default.
+
+Example:
+
+Sort test path alphabetically
+
+```js
+const Sequencer = require('@jest/test-sequencer').default;
+
+class CustomSequencer extends Sequencer {
+  sort(tests: Array<Test>) {
```

Comment:
Shouldn't use TS here I think, people probably expect this to work when copy-pasted